src/Meta/Utils.cpp

```diff
@@ -117,16 +117,20 @@ bool isAlpha(const std::vector<std::string>& strings, size_t index)
     return true;
 }
 
-std::string Utils::getCommonWordPrefix(const std::vector<std::string>& strings)
+std::string Utils::calculateEnumFieldsPrefix(std::string enumName, std::vector<std::string> fields)
```

Comment:
I figure it out how to make them `const` and passed by reference. I will update the branch.